### PR TITLE
Trivial: Fix typos in BIP-118

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

File: bip-0118.mediawiki [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
 == Abstract ==
 This BIP describes a new signature hash flag (<tt>sighash</tt>-flag) for
-segwit transactions. It removes any commitment to the output being
+segwit 
+ns. It removes any commitment to the output being
```
Comment (luke-jr) [inline bip-0118.mediawiki]: This doesn't look like a typo fix...

**Comment by luke-jr** [comment]
@cdecker

File: bip-0118.mediawiki (hunk 1 above)
Comment (erikarvstedt) [reply bip-0118.mediawiki]: Yeah, sorry, the Github in-line editor messed it all up.
I've pushed a fixed version.

**Comment by cdecker** [comment]
LGTM :+1:

**Pull request merged**